pandas/tslib.pyx

```diff
@@ -656,6 +651,46 @@ for field in fields:
     prop = property(fget=lambda self: np.nan)
     setattr(NaTType, field, prop)
 
+def _make_error_func(func_name):
```

Comment:
can you add a little explanation of what is going on here (for future readers) ,and put the issue number as a comment as well